### ISSUE TITLE
Consider naming scheme for methods on the api client

### ISSUE THREAD

**Issue by CottageDwellingCat** [Nexfinity/Fluxer.Net#7]
We might want to make a common naming scheme for methods in the api client, eg:`[Method][Object]<Note>` (`PatchChannelMessage`, `PostTokenRevoce`) however some endpoints are not clear (do new users know what a channel message is? Could anyone figure out what the Token endpoint did without looking it up or asking someone?)

**Comment by MrCakeSlayer** [comment]
I agree with the common naming and your proposed method should be fine for that.
As for your question of: `Could anyone figure out what the Token endpoint did without looking it up or asking someone`
For starters we can use summary docs on each endpoint (as well as other code to clarify this) plus we have an actual documentation website users will be able to reference which should mostly solve this.
